src/containers/Storage/Storage.js

```diff
@@ -80,6 +79,8 @@ class Storage extends React.Component {
                 filter: FILTER_OPTIONS.All,
                 nodeId,
                 type: storageType,
+            }, {
+                concurrentId: 'getStorageInfo',
```

Comment:
Let's make this method as a separate function to avoid code duplication.

Reply:
what method are you talking about?